fbw-a32nx/src/systems/fmgc/src/flightplanning/test/FlightPlan.ts

```diff
@@ -22,7 +27,7 @@ export function dumpFlightPlan(plan: BaseFlightPlan): string {
       }
 
       const isActive = index === plan.activeLegIndex;
-      const isMissedApproach = index > plan.destinationLegIndex;
+      // const isMissedApproach = index > plan.destinationLegIndex;
```

Comment:
Do we need to have this commented out or can we remove it entirely?

Reply:
I commented it out because it's not used (because of the commented code below calling `chalk` which I removed at some point). This is part of some flight plan test code that was only needed during FMSv2 development, so I will remove this entirely.